client/shipping-label/views/purchase/steps/packages/list.js

```diff
@@ -44,7 +48,7 @@ const OrderPackages = ( { packages, updateWeight, dimensionUnit, weightUnit, err
 					className="wcc-package-weight"
```

Comment:
Better to change the ID of this field from `weight_${pckgIndex}` to `weight_${pckgId}`